packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -102,26 +103,23 @@ export default function ActionButtons({
             }
           >
             <Button
-              className="pointer-events-auto hover:text-theme-color-avocado"
+              className={classNames(
+                'pointer-events-auto',
+                post?.numUpvotes > 0 ? '!pl-1 !pr-3' : '!px-1',
+              )}
               id={`post-${post.id}-upvote-btn`}
               color={ButtonColor.Avocado}
-              icon={
-                <UpvoteIcon
-                  secondary={post?.userState?.vote === UserPostVote.Up}
-                />
-              }
               pressed={post?.userState?.vote === UserPostVote.Up}
               onClick={() => onUpvoteClick?.(post)}
               variant={ButtonVariant.Tertiary}
             >
+              <UpvoteIcon
+                secondary={post?.userState?.vote === UserPostVote.Up}
+                size={IconSize.Medium}
+              />
               {post?.numUpvotes > 0 ? (
                 <InteractionCounter
-                  className={classNames(
-                    '!min-w-[2ch] font-bold tabular-nums typo-callout',
-                    post?.userState?.vote === UserPostVote.Up
-                      ? 'text-theme-color-avocado'
-                      : 'text-theme-label-tertiary',
-                  )}
+                  className="ml-1.5 !min-w-[2ch] tabular-nums"
```

Comment:
Aren't the colors and typo-callout needed anymore?

Reply:
No, since the styles are now applied from the `Button` component itself now.